docs/src/styles/docs/sidebar.scss

```diff
@@ -1,55 +1,160 @@
+@import './_variables.scss';
+
+
+@keyframes showSidebar {
+  0% {
+    transform: translate(-100%);
+  }
+  100% {
+    transform: translate(0);
+  }
+}
+
+@keyframes showOverlay {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
 .docs-sidebar {
-  display: none;
+  display: block;
   
-  @media (min-width: 62rem) {
-    display: block;
+  &.collapsed {
+    display: none;
+    @media (min-width: $breakpoint-large) {
+      display: block;
+    }
+  }
+}
+
+.docs-sidebar-spacer {
+  width: 0;
+  
+  // desktop
+  @media (min-width: $breakpoint-large) {
     flex-shrink: 0;
-    width: 16rem;
+    width: 20rem;
     position: relative;
+    
+    .docs-home & {
+      width: 0;
+    }
+  }
+}
+
+.docs-framework-link {
+  justify-content: flex-start;
+  border-color: var(--amplify-colors-border-secondary);
+  gap: var(--amplify-space-xs);
+  
+  &.current {
+    background-color: var(--amplify-components-button-hover-background-color);
+    border-color: var(--amplify-components-button-active-border-color);
+  }
+}
+
+.docs-sidebar-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  opacity: 0;
+  background-color: var(--amplify-colors-overlay-40);
+  
+  .expanded > & {
+    width: 100vw;
+    height: 100vh;
+    opacity: 1;
+    animation: showOverlay var(--amplify-time-long) forwards;
+  }
+  
+  @media (min-width: $breakpoint-large) {
+    .expanded > & {
+      display: none;
+    }
+    
+    .docs-home .expanded > & {
+      display: block;
+    }
   }
 }
 
+
 .docs-sidebar-inner {
-  height: calc(100vh - var(--docs-header-height));
-  width: 16rem;
+  // For mobile screens, take up 90% width
+  --width: 90vw;
+  // Tablet and up use a fixed width
+  @media (min-width: $breakpoint-small) {
+    --width: 20rem;
+  }
+  
+  // This is to get the secondary link text lined up with the primary nav item
+  --secondary-nav-indent: calc(1.25rem + var(--amplify-space-small) + var(--amplify-space-small));
+  
+  // Custom expander styles
+  --amplify-components-expander-box-shadow: none;
+  --amplify-components-expander-item-box-shadow: none;
+  --amplify-components-expander-header-box-shadow: none;
+  --amplify-components-expander-trigger-padding-inline-start: 0;
+  --amplify-components-expander-trigger-padding-inline-end: 0;
+  --amplify-components-expander-trigger-min-height: 0;
+  --amplify-components-expander-item-border-start-start-radius: 0;
+  --amplify-components-expander-item-border-start-end-radius: 0;
+  --amplify-components-expander-item-border-end-start-radius: 0;
+  --amplify-components-expander-item-border-end-end-radius: 0;
+  --amplify-components-expander-item-focus-box-shadow: none;
```

Comment:
I think this removed the focus style from our expanders, and it looks like the child elements are picking up the default focus ring (which gets cut off). Maybe we can add a different type of focus style from our usual expander styles?

<img width="295" alt="Screen Shot 2022-05-25 at 6 12 29 PM" src="https://user-images.githubusercontent.com/376920/170375920-33ee5691-9944-4ae6-9194-b39a3457bbd6.png">

Reply:
fixed now